<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BooleanEdit</class>
 <widget class="QWidget" name="BooleanEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ActionTools::CodeComboBox" name="comboBox"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="ActionTools::SwitchTextModePushButton" name="switchTextModePushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionTools::SwitchTextModePushButton</class>
   <extends>QPushButton</extends>
   <header>actiontools/switchtextmodepushbutton.hpp</header>
  </customwidget>
  <customwidget>
   <class>ActionTools::CodeComboBox</class>
   <extends>QComboBox</extends>
   <header>actiontools/codecombobox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
